pkg/search/proxy/framework/plugins/karmada/karmada.go

```diff
@@ -0,0 +1,76 @@
+package karmada
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"path"
+
+	restclient "k8s.io/client-go/rest"
+
+	"github.com/karmada-io/karmada/pkg/search/proxy/framework"
+	"github.com/karmada-io/karmada/pkg/util/proxy"
+)
+
+const (
+	order = 3000
```

Comment:
what means order = 3000? give some comments?

Reply:
It's just a number for sort plugins. There is no special meaning.

Reply:
I think users should look documents before `Plugin.Order()`.

Reply:
I give a comment about why I keep a big gap between in tree plugins. Rather than use 1,2,3.

Reply:
Also give some comments like: the smaller the number is, the higher priority it is.

Reply:
This comment is already in `Plugin.Order()`. I don't want to repeat myself in 3 places.